vc/vc.go

```diff
@@ -415,6 +415,9 @@ func CreateJWTVerifiableCredential(ctx context.Context, template VerifiableCrede
 		// so a template using ValidFrom/ValidUntil would not match the final VC
 		return nil, errors.New("cannot use validFrom/validUntil to generate JWT-VCs")
 	}
+	if template.CredentialStatus != nil {
+		claims["vc"].(map[string]any)["credentialStatus"] = template.CredentialStatus
```

Comment:
move "vc" out of the initilization above, then you don't need complicated-looking casting